redisvl/extensions/session_manager/semantic_session.py

```diff
@@ -51,6 +52,7 @@ def __init__(
             redis_url (str, optional): The redis url. Defaults to redis://localhost:6379.
             connection_kwargs (Dict[str, Any]): The connection arguments
                 for the redis client. Defaults to empty {}.
+            dtype (str): The data type for the prompt vector. Defaults to "float32".
```

Comment:
Should this be in the docstring? if so, should be in all extension __init__ docstrings

Reply:
I pulled it from the doc string as it's not a named arg, but we probably should have some way to let users know this is an option.
Maybe include it in the docs and/or notebook examples?